src/components/Form/Form.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {Button} from '../Button';
+import {MultiSelect} from '../MultiSelect/';
+
+import styles from './Form.module.css';
+
+interface FormProps {
+    initialValues?: FormValues;
+}
+
+interface FormValues {
+    title: string;
+    date: string;
+    url: string;
+    genre: string[];
+    overview: string;
+    runtime: string;
+}
+
+export const Form = ({
+    initialValues
+}: FormProps) => {
+    const [formData, setFormData] = React.useState<FormValues>(initialValues ?? {} as FormValues);
+
+    const handleTitleChange = (value) => {
```

Comment:
You can create some helper function:
```js
const changeForm = (name, value) => {
  setFormData(prevState => ({...prevState, [name]: value}))
}
```

and then use it like this:
```js
changeForm('title', value)
```

Reply:
This is done